### ISSUE TITLE
请问resnet50 kl loss 256*192配置

### ISSUE THREAD

**Issue by zonghaofan** [leeyegy/SimDR#9]
我用resnet50 kl loss 256*192  sigma=1训练出来ap是0.685，没有文章0.71高。

**Comment by leeyegy** [comment]
Could you provide more details you are using? The recommended setting is in experiments/coco/resnet/simdr/original/nmt_res50_256x192_d256x3_adam_lr1e-3_deconv3_split2.yaml. The parameter **sigma** should do nothing with the proposed method.

**Comment by zonghaofan** [comment]
> Could you provide more details you are using? The recommended setting is in experiments/coco/resnet/simdr/original/nmt_res50_256x192_d256x3_adam_lr1e-3_deconv3_split2.yaml. The parameter **sigma** should do nothing with the proposed method.

AUTO_RESUME: true
CUDNN:
  BENCHMARK: true
  DETERMINISTIC: false
  ENABLED: true
DATA_DIR: ''
GPUS: (0,1,2,3)
OUTPUT_DIR: 'output'
LOG_DIR: 'log'
WORKERS: 24
PRINT_FREQ: 20

DATASET:
  COLOR_RGB: false
  DATASET: 'coco'
  ROOT: '/share/public/coco-icp'
  TEST_SET: 'val2017'
  TRAIN_SET: 'train2017'
  FLIP: true
  ROT_FACTOR: 40
  SCALE_FACTOR: 0.3
MODEL:
  NAME: 'pose_resnet_upfree'
  SIMDR_SPLIT_RATIO: 2.0
  HEAD_INPUT: 1344
  PRETRAINED: '/data/pretrained/imagenet/resnet50-19c8e357.pth'
  IMAGE_SIZE:
  - 192
  - 256
  HEATMAP_SIZE:
  - 192
  - 256
  SIGMA: 1
  NUM_JOINTS: 17
  COORD_REPRESENTATION: 'sa-simdr'
#  COORD_REPRESENTATION: 'simdr'
  EXTRA:
    FINAL_CONV_KERNEL: 1
    CHANNEL_PER_JOINT: 28
    NUM_LAYERS: 50
LOSS:
  USE_TARGET_WEIGHT: true
  TYPE: 'KLDiscretLoss'
#  TYPE: 'NMTCritierion'
TRAIN:
  BATCH_SIZE_PER_GPU: 32
  SHUFFLE: true
  BEGIN_EPOCH: 0
  END_EPOCH: 140
  OPTIMIZER: 'adam'
  LR: 0.001
  LR_FACTOR: 0.1
  LR_STEP:
  - 90
  - 120
  WD: 0.0001
  GAMMA1: 0.99
  GAMMA2: 0.0
  MOMENTUM: 0.9
  NESTEROV: false
TEST:
  BATCH_SIZE_PER_GPU: 32
  COCO_BBOX_FILE: '/share/public/coco-icp/annotations/COCO_val2017_detections_AP_H_56_person.json'
  BBOX_THRE: 1.0
  IMAGE_THRE: 0.0
  IN_VIS_THRE: 0.01
  MODEL_FILE: '/home/fanzonghao/AI/papercode/SimDR/pretrained/pose_upfree_resnet_50_256x192_cpj28.pth'
  NMS_THRE: 1.0
  OKS_THRE: 0.9
  FLIP_TEST: false
  POST_PROCESS: false
  SHIFT_HEATMAP: true
  USE_GT_BBOX: true
DEBUG:
  DEBUG: true
  SAVE_BATCH_IMAGES_GT: true
  SAVE_BATCH_IMAGES_PRED: true
  SAVE_HEATMAPS_GT: true
  SAVE_HEATMAPS_PRED: true

**Comment by leeyegy** [comment]
The paper result 71.4 AP can be reproduced by using the recommended setting(experiments/coco/resnet/simdr/original/nmt_res50_256x192_d256x3_adam_lr1e-3_deconv3_split2.yaml), which keeps the three-layers deconvolutional modules.

**Comment by leeyegy** [comment]
It seems that the yaml file you are using has beed changed a lot.
For example, the FLIP_TEST should be set as TRUE, which may affect the performance. 
AND, if you want to reproduce the 70.8 AP(the version without any deconvolutional module), we recommend you to follow the setting in experiments/coco/resnet/simdr/upsample_free/nmt_res50_256x192_d256x3_adam_lr1e-3_split_2_cpj28.yaml

**Comment by zonghaofan** [comment]
> It seems that the yaml file you are using has been changed a lot. For example, the FLIP_TEST should be set as TRUE, which may affect the performance. AND, if you want to reproduce the 70.8 AP(the version without any deconvolutional module), we recommend you to follow the setting in experiments/coco/resnet/simdr/upsample_free/nmt_res50_256x192_d256x3_adam_lr1e-3_split_2_cpj28.yaml

ok thanks

**Comment by leeyegy** [comment]
Feel free to reopen this issue, if there is any problems in reproducing.

**Comment by atonyo11** [comment]
@leeyegy, @fanpeace  Sorry for bothering you. As I see in the "experiments/coco/resnet/simdr/original/nmt_res50_256x192_d256x3_adam_lr1e-3_deconv3_split2.yaml". Loss type is NMTCritierion, not KLDiscretLoss? Does NMTCritierion is a type of Kullback–Leibler divergence loss? Thank!
